<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nowyKolorDialog</class>
 <widget class="QDialog" name="nowyKolorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj kolor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Kolor Nr:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="spinBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditSk1">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditSk2">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditSk3">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sk dominujÄ…cy:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditSkDominujacy">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>99</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Dodaj klienta</string>
       </property>
       <property name="text">
        <string>Sk 1:</string>
       </property>
       <property name="icon">
        <iconset resource="zasoby.qrc">
         <normaloff>:/zasoby/add.png</normaloff>:/zasoby/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>26</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>99</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Dodaj klienta</string>
       </property>
       <property name="text">
        <string>Sk 2:</string>
       </property>
       <property name="icon">
        <iconset resource="zasoby.qrc">
         <normaloff>:/zasoby/add.png</normaloff>:/zasoby/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>26</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>99</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Dodaj klienta</string>
       </property>
       <property name="text">
        <string>Sk 3:</string>
       </property>
       <property name="icon">
        <iconset resource="zasoby.qrc">
         <normaloff>:/zasoby/add.png</normaloff>:/zasoby/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>26</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBox</tabstop>
  <tabstop>lineEditSk1</tabstop>
  <tabstop>lineEditSk2</tabstop>
  <tabstop>lineEditSk3</tabstop>
  <tabstop>lineEditSkDominujacy</tabstop>
 </tabstops>
 <resources>
  <include location="zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
